### DISCUSSION TITLE
Poster art for advertised application

### DISCUSSION THREAD

**Discussion by salty2011** [LizardByte/community#10] [Sunshine]
When using moonlight with sunshine all advertised applications currently get a placeholder poster art. However moonlight client is capable of displaying post art as long as the server defines the art.

Suggested feature is to add the capability for Sunshine server to allow defining artwork for the published application to the client.

**Comment by PapyKahan** [comment]
<img width="954" alt="image" src="https://user-images.githubusercontent.com/9347985/150359498-2b93987e-ed9f-4a03-a31a-26796f54869d.png">

Work in progress. Implemation of configuration from WebUI needs to be done. The backend is working.

**Comment by salty2011** [comment]
Nice work, happy to test. I assume when you say backend you mean addition settings in the Config?

**Reply by PapyKahan** [reply]
Yes exactly, but now everything seems to work. I'm doing minor refactoring and more test before submiting my pull request.

**Comment by ReenigneArcher** [comment]
Artifacts here from last PR workflow if you want to test.

https://github.com/SunshineStream/Sunshine/actions/runs/1724663053

**Comment by ReenigneArcher** [comment]
This has been added and will be available in the next release.
